src/transit_data/data_update.rs

```diff
@@ -252,18 +252,20 @@ where
         BoardTimes: Iterator<Item = SecondsSinceTimezonedDayStart> + ExactSizeIterator + Clone,
         DebarkTimes: Iterator<Item = SecondsSinceTimezonedDayStart> + ExactSizeIterator + Clone,
     {
-        // let's check that
-        //  - the base vehicle does not exists
-        //  - the real time vehicle does not exists
-        if self
-            .vehicle_journey_to_timetable
-            .base_vehicle_exists(&vehicle_journey_idx)
-        {
-            return Err(InsertionError::BaseVehicleJourneyAlreadyExists(
-                vehicle_journey_idx.clone(),
-            ));
+        // if we add on the base level, let's check that
+        // the vehicle does not exists in base vehicles
+        if real_time_level == RealTimeLevel::Base {
```

Comment:
I guess it would change large part of the code, but here is the vocabulary mainly used in Kraken
```suggestion
        if data_freshness_level == DataFreshnessLevel::Base {
```

Reply:
yeah, I think I'll keep RealTimeLevel